packages/pfsense/data_stream/log/elasticsearch/ingest_pipeline/default.yml

```
@@ -61,7 +64,7 @@ processors:
       if: ctx.event.provider == 'charon'
   - pipeline:
       name: '{{ IngestPipeline "dhcp" }}'
-      if: ctx.event.provider == 'dhcpd'
+      if: '["dhcpd", "dhclient", "dhcp6c"].contains(ctx?.event?.provider)'
```

We know that `ctx` is non-null and also that `ctx.event` is non-null (due to `event.kind` and `event.original` above).

```suggestion
      if: '["dhcpd", "dhclient", "dhcp6c"].contains(ctx.event.provider)'
```